spec/acceptance/rvm_system_spec.rb

```diff
@@ -111,8 +112,8 @@ class { 'rvm': } ->
 
     it 'reflects installed rubies' do
       shell('/usr/local/rvm/bin/rvm list') do |r|
-        r.stdout.should =~ Regexp.new(Regexp.escape("\n   #{ruby19_version}"))
-        r.stdout.should =~ Regexp.new(Regexp.escape("\n   #{ruby20_version}"))
+        r.stdout.should =~ Regexp.new(Regexp.escape(ruby27_version))
+        r.stdout.should =~ Regexp.new(Regexp.escape(ruby26_version))
```

Comment:
Also, much easier is to just use `String.include?` instead of using a regex:
```suggestion
        expect(r.stdout).to include(ruby27_version)
        expect(r.stdout).to include(ruby26_version)
```

Though the old code was checking that it was at the start of a line with some indenting. That could be more explicit.